ajax/libs/bottlejs/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "bottlejs",
+  "filename": "bottle.min.js",
+  "version": "1.4.0",
+  "description": "A powerful dependency injection micro container",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/young-steveo/bottlejs.git"
+  },
+  "keywords": [
+    "di",
+    "dependency",
+    "injection",
+    "micro",
+    "angular",
+    "pimple"
+  ],
+  "author": "Stephen Young <young.steveo@gmail.com>",
+  "license": "MIT",
+  "npmName": "bottlejs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
We need to make sure we won't fetch this file type. Eventhough this type ignore by `.gitignore`.